Map.h

```diff
@@ -0,0 +1,62 @@
+/**
+* \class Map
+*
+* \brief Map class
+*
+* This class's role is that it gets a map from a file. In this map the '#' character indicates the wall, and a ' ' indicates the free space, where the Character can move.
+*
+* \author skrobi12, hajdunorbi, Szabi1104
+*
+* \version 7.0
+*
+* \date 2020/11/22 19:17
+*/
+
+
+#ifndef MAP_H
+#define MAP_H
+
+#include <string>
+#include <fstream>
+#include <vector>
+
+class Map{
+	std::vector<std::string> map;		///< This is a vector type variable, which contains the map of the game.
```

Comment:
Nem azért (is) van a type nevű enum, hogy abból legyen 2D-s tárolás? Szerintem a következő résznél lesz átláthatatlanabb így, de lehet ez is jó.

Reply:
Hát mi így gondoltuk, egyenlőre nekünk működik, aztán lehet később átírásra kerül, mert, ahogy fogalamztál átláthatalan lesz a kód :)